src/CommentBox.js

```diff
@@ -252,9 +270,11 @@ class CommentBox extends EventEmitter {
         this.postEl = containerEl.querySelector(constants.SELECTOR_ANNOTATION_BUTTON_POST);
 
         // Add event listeners
+        this.textAreaEl.addEventListener('focus', this.focus);
         this.cancelEl.addEventListener('click', this.onCancel);
         this.postEl.addEventListener('click', this.onPost);
         if (this.hasTouch) {
+            this.textAreaEl.addEventListener('keydown', this.focus);
```

Comment:
Does focus not fire on keydown in a mobile setting?

Reply:
nope

Reply:
I guess the focus event of clicking on the text area fires first usually